src/main/java/eu/bbmri_eric/negotiator/negotiation/NegotiationServiceImpl.java

```diff
@@ -262,6 +265,10 @@ public Iterable<NegotiationDTO> findByFiltersForUser(
   @Override
   public Iterable<NegotiationDTO> findAllForNetwork(
       Long networkId, NegotiationFilterDTO filtersDTO) {
+    if (!personRepository.isNetworkManager(
```

Comment:
shouldn't this be delegated to the new NegotiationAccessManager?

Reply:
hmm... this is applicable only for networks, so maybe either leave it as it is or create a NetworkAccessManger?